clients/src/main/java/org/apache/kafka/common/requests/AlterShareGroupOffsetsRequest.java

```diff
@@ -78,6 +78,24 @@ public static AlterShareGroupOffsetsRequest parse(Readable readable, short versi
         );
     }
 
+    public static AlterShareGroupOffsetsResponseData.AlterShareGroupOffsetsResponseTopic getErrorAlterShareGroup(
```

Comment:
in AK `get` prefix on methods is generally avoided. You could perhaps use the infinitely more ubiquitous name - just `from`

```
public static AlterShareGroupOffsetsResponseData.AlterShareGroupOffsetsResponseTopic from(Errors error)
```